tsconfig.json

```diff
@@ -56,7 +56,7 @@
     // "sourceRoot": "",                      /* Specify the location where debugger should locate TypeScript files instead of source locations. */
     // "mapRoot": "",                         /* Specify the location where debugger should locate map files instead of generated locations. */
     // "inlineSourceMap": true,               /* Emit a single file with source maps instead of having a separate file. */
-    // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
+    // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file requires '--inlineSourceMap' or '--sourceMap' to be set. */
```

Comment:
what happened here haha

Reply:
my IDE would inject semicolons every time i refactored code into a separate file, so i did a global replacement of the `;` character lol